subprojects/core/src/main/java/org/gradle/configuration/project/LifecycleProjectEvaluator.java

```diff
@@ -69,12 +67,12 @@ private void doConfigure(ProjectInternal project, ProjectStateInternal state) {
         } finally {
             state.setExecuting(false);
             state.executed();
-            notifyAfterEvaluate(listener, project, state);
+            buildOperationExecutor.run(new AfterBeforeEvaluateHooks(project, state));
         }
     }
 
-    private void notifyAfterEvaluate(ProjectEvaluationListener firstBatch, final ProjectInternal project, final ProjectStateInternal state) {
-        @Nullable ProjectEvaluationListener nextBatch = firstBatch;
+    private void notifyAfterEvaluate(final ProjectInternal project, final ProjectStateInternal state) {
+        @Nullable ProjectEvaluationListener nextBatch = project.getProjectEvaluationBroadcaster();
```

Comment:
not part of your changeset. but is there value in having the `@Nullable` here? I don't see we use this annotation in other places like that

Reply:
I've removed it.